lrpar/src/lib/ctbuilder.rs

```diff
@@ -1145,7 +1151,7 @@ where
                 "    // {rulename}
     #[allow(clippy::too_many_arguments)]
     #[allow(unsafe_code)] // Allow an action to embed unsafe blocks within it.
-    fn {prefix}action_{}<'lexer, 'input: 'lexer>({prefix}ridx: ::cfgrammar::RIdx<{storaget}>,
+    pub fn {prefix}action_{}<'lexer, 'input: 'lexer>({prefix}ridx: ::cfgrammar::RIdx<{storaget}>,
```

Comment:
Does this need to be `pub`? I'd ideally prefer not to expose it to the user (but if we have to, we have to).

Reply:
I'm pretty sure it needs some visibility, you're right though `pub` is probably too much I think it can just be `pub(super)`.
In reality though even though it is `pub`, because the `mod` isn't pub, and we aren't `pub use` it should only be visible to `super`.

But it's better to spell these things out with `pub(super)` I think.